specs/deneb/beacon-chain.md

```diff
@@ -24,6 +24,7 @@
 - [Helper functions](#helper-functions)
   - [Misc](#misc)
     - [`kzg_commitment_to_versioned_hash`](#kzg_commitment_to_versioned_hash)
+  - [Modified `get_attestation_participation_flag_indicies`](#modified-get_attestation_participation_flag_indicies)
```

Comment:
In the altair spec, `get_attestation_participation_flag_indices` is under `Beacon state accessors`, so I think we should add a topic of `Beacon state accessors` in this spec as well.